pallets/committee/src/lib.rs

```diff
@@ -147,6 +154,8 @@ pub mod pallet {
 	#[pallet::genesis_build]
 	impl<T: Config> GenesisBuild<T> for GenesisConfig<T> {
 		fn build(&self) {
+			VotingPeriod::<T>::set(T::VotingPeriod::get());
```

Comment:
Kept the old `VotingPeriod` in runtime config for the initial `VotingPeriod`

Reply:
this is fine for genesis.